src/ExaPF.jl

```diff
@@ -24,14 +24,18 @@ using TimerOutputs
 export solve
 
 # Import submodules
-include("parse/parse.jl")
-using .Parse
 include("ad.jl")
 using .AD
 include("algorithms/precondition.jl")
 using .Precondition
+include("indexes.jl")
+using .IdxSet
```

Comment:
I would be in favor of choosing a more explicit name for this module. How about `IndexSet`? `IndexingSet`?

Reply:
Fixed